state/futures/selectors.ts

```diff
@@ -338,8 +352,10 @@ export const selectIsolatedMarginTradeInputs = createSelector(
 	(side, tradeInputs) => {
 		const inputs = unserializeIsolatedMarginTradeInputs(tradeInputs);
 		const deltas = {
-			susdSizeDelta: side === PositionSide.LONG ? inputs.susdSize : inputs.susdSize.neg(),
-			nativeSizeDelta: side === PositionSide.LONG ? inputs.nativeSize : inputs.nativeSize.neg(),
+			susdSizeDelta:
+				side === PositionSide.LONG ? inputs.susdSize.abs() : inputs.susdSize.abs().neg(),
```

Comment:
Just an fyi `size` is always positive so shouldn't need this change. Only sizeDelta should reflect positive or negative.